src/components/UI/MovieCategoryDropdown.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="dropdown-categories">
+    <label class="dropdown-categories__label" for="">Categories</label>
+    <select
+      class="dropdown-categories__select"
+      @change="$store.commit('setSelectedGenre', $event.target.value)"
+    >
+      <option
+        class="dropdown-categories__options"
+        v-for="genre in ['All Categories', ...$store.getters.genreList]"
```

Comment:
While this is absolutely correct and working - it might be a good idea to make a `computed` out of this:
i.e.

`v-for="genre in categories"`

and then have a computed like following:

```
computed: {
  categories () {
    return [
      'All categories',
      ...this.$store.getters.genreList
    ];
  }
}
```

This way if we would need to for some reason also access this somewhere else in our component - we have it available everywhere 💡 

As another cool tip - have you heard of [`mapGetters`](https://v3.vuex.vuejs.org/guide/getters.html#the-mapgetters-helper)? We could use it to simplify and clean up our code even more 🚀

We could use `mapGetters` to have the `genreList` as another computed. For example expanding on the idea above:

```
computed: {
  // this will give us a 'genreList' computed
  ...mapGetters([
    'genreList'
  ]),

  categories () {
    return [
      'All categories',
      ...this.genreList
    ];
  }
}
```

Reply:
Hi Szymon,

Thank you very much for this tip. I have never thought about moving this to computed and this is now not only reusable but also reads so much cleaner 😸 
I have not event looked at mapGetters (during training it was mentioned that we might be better off not using them for this project UNLESS I REMEMBER INCORRECTLY) but thanks to you explaining it nicelly with the example I think I will use them more often now - they are so much cleaner!!! ❤️